.github/workflows/Coverage.yml

```diff
@@ -0,0 +1,93 @@
+name: Coverage
+
+on:
+  workflow_call:
+
+jobs:
+
+
+#
+# pyGHDL Coverage (MINGW64)

```

Comment:
Can we reused actions from `pyTooling/actions`?

I think running coverage can be on Linux or Windows. Maybe we should extend `pyTooling/actions` to support Windows too.

Reply:
Agree. However, let's keep it out of this PR, since I cannot deal with it now.
After this, split, we can focus on the similariries between Coverage.yml and pyTooling Actions.